realm/realm-library/src/main/java/io/realm/internal/Property.java

```diff
@@ -73,9 +73,9 @@
         this(nativeCreatePersistedProperty(name, convertFromRealmFieldType(type, isRequired), isPrimary, isIndexed));
     }
 
-    Property(String name, RealmFieldType type, @Nullable String linkedClassName) {
+    Property(String name, RealmFieldType type, @Nullable String linkedClassName, boolean required) {
         // Ignore the isRequired when creating the linking property.
```

Comment:
this comment can be removed

Reply:
I missed this comment. I'll remove this comment in part 9